convert/src/file.ts

```diff
@@ -7,24 +7,23 @@ const base: { open: string, close: string } = { open: "", close: "" };
 const twClass = "Tailwind";
 
 base.open = `// http://github.com/dvkndn/typed-tailwind
-export const Tw = (): ${twClass} => new ${twClass}();
+export const Tw = new ${twClass}();
```

Comment:
This is safe to create a new `Tw` immediately b/c all of the `.foo()` methods make copies/new `Tailwind` instances, i.e. each `Tailwind` instance is immutable.